### PR TITLE
Update readme.md

### PR DESCRIPTION
**Pull request by nachoiacovino**
Much needed change to update the Readme.

Added utm links, add a one click deployment button and a link to our support page.

Also, we converted the repo to a template repo for an easier DX.

I also deleted CLERK_API_KEY from .env.local.sample since it wasn't necessary.

### CODE REVIEW

**Review by SokratisVidros — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,34 +1,28 @@
-This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
+# Clerk - Next.js starter repository
 
-## Getting Started
+This repository shows an example use case for how you can authenticate your users using [Clerk](https://www.clerk.dev/?utm_source=github&utm_medium=starter_repos&utm_campaign=nextjs_starter) as an authentication provider on your Next.js application.
```
Comment (SokratisVidros) [inline README.md:3]:
```suggestion
This repository shows an example about how to add user authenticating to your Next.js application using [Clerk](https://www.clerk.dev/?utm_source=github&utm_medium=starter_repos&utm_campaign=nextjs_starter).
```

File: README.md [hunk 2]
```diff
@@ -1,34 +1,28 @@
...
-# or
-yarn dev
-```
+You have two option to run the starter, we recommend the one click deployment with Vercel. This way, environment variables are set up for you automatically and it's a much easier experience.
 
-Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.
+### Option 1 - One click deployment with Vercel
 
-You can start editing the page by modifying `pages/index.js`. The page auto-updates as you edit the file.
+You can use the following deployment button from Vercel, [this article on our blog](https://clerk.dev/blog/clerk-vercel-one-click-deployment?utm_source=github&utm_medium=starter_repos&utm_campaign=nextjs_starter) guides you step to step on how to do so.
```
Comment (SokratisVidros) [inline README.md:11]:
```suggestion
You can use the following deployment button from Vercel. For more detailed read our [official step by step guide](https://clerk.dev/blog/clerk-vercel-one-click-deployment?utm_source=github&utm_medium=starter_repos&utm_campaign=nextjs_starter).
```

File: README.md [hunk 3]
```diff
@@ -1,34 +1,28 @@
...
 
-The `pages/api` directory is mapped to `/api/*`. Files in this directory are treated as [API routes](https://nextjs.org/docs/api-routes/introduction) instead of React pages.
+### Option 2 - Run manually
 
-## Learn More
+To run the example locally you need to:
 
-To learn more about Next.js, take a look at the following resources:
+1. Click [**"Use this template"**](https://github.com/clerkinc/clerk-nextjs-starter/generate) to create a copy of this repository on your GitHub account.
+2. Clone your repository locally.
```
Comment (SokratisVidros) [inline README.md:20]:
```suggestion
2. Clone your repository.
```

File: README.md [hunk 4]
```diff
@@ -1,34 +1,28 @@
...
+3. Sign up for a Clerk account at [Clerk.dev](https://www.clerk.dev/?utm_source=github&utm_medium=starter_repos&utm_campaign=nextjs_starter).
```
Comment (SokratisVidros) [inline README.md:21]:
```suggestion
3. Sign up at [Clerk.dev](https://www.clerk.dev/?utm_source=github&utm_medium=starter_repos&utm_campaign=nextjs_starter).
```

File: README.md [hunk 5]
```diff
@@ -1,34 +1,28 @@
...
+4. Setup the required Clerk Frontend API variable from your Clerk project as shown at [the example env file](./.env.local.sample).
```
Comment (SokratisVidros) [inline README.md:22]: We should add a step to describe that we need to create an instance.

**Review by SokratisVidros — CHANGES_REQUESTED** [review]

File: README.md (hunk 5 above)
Comment (nachoiacovino) [reply README.md]: Added a step!

**Pull request merged**